template/project.json

```diff
@@ -1,20 +1,29 @@
 {
-    "version": "1.0.0-*",
-    "compilationOptions": {
-        "emitEntryPoint": true
-    },
-
-    "compilerName": "fsc",
-    "compileFiles": [
-        "Program.fs"
-    ],
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "compilerName": "fsc",
+  "compileFiles": [
+    "Program.fs"
+  ],
+  "dependencies": {
+    "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160329",
+    "Microsoft.NETCore.App": {
+      "version": "1.0.0-rc2-23931"
+    }
+  },
 
-    "dependencies": {
-        "Microsoft.FSharp.Core.netcore": "1.0.0-alpha-160318",
-        "NETStandard.Library": "1.5.0-rc2-23911"
-    },
+  "tools": {
+       "dotnet-compile-fsc": "1.0.0-*"
+   },
 
-    "frameworks": {
-        "dnxcore50": { }
+  "frameworks": {
+    "netstandard1.5": {
+      "imports": [
+        "portable-net45+win8",
+        "dnxcore50"
+      ]
```

Comment:
When I try using `netstandar1.5` instead of `dnxcore50`, I get `/root/template/unknown(1,1): error FS0078: Unable to find the file 'mscorlib.dll' in any of`.